src/style.css

```diff
@@ -0,0 +1,48 @@
+body {
+  margin: 0;
+  --clear-star: url(images/star.svg);
+  --golden-star: url(images/star-active.svg);
+}
+
+.stars {
+  display: flex;
+  height: 15px;
+}
+
+.star {
+  width: 15px;
+  margin-right: 4px;
+  background: var(--clear-star);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars.stars--1 :not(:nth-child(n+2)) {
+  background: var(--golden-star);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
you've already add background-repeat and background-position to each star above, remove these styles everywhere else
```suggestion
```

Reply:
I had did it before in commit https://github.com/mate-academy/layout_stars/pull/409/commits/0d6828ddb3a3a627044a06f644a02d7f65ce0bb9
but I had unexpected results
https://drive.google.com/file/d/1Xx0hwF9UExtooLIGtk6HrHj22hj4Npum/view?usp=sharing
as I understand it is wrong to make a gap for background-image and other background properties in different css classes
Am I right?